### PR TITLE
Add security group tags to admin section and refresh data

### PR DESCRIPTION
**Pull request by michaelbish**
Admin section now allows users to assign security groups to their reports

**Settings data will need to be reset as the structure of the AadGroups column has changed.**

### CODE REVIEW

File: src/PowerBiAuditApp.Client/Controllers/AdminController.cs [hunk 1]
```diff
@@ -45,6 +46,9 @@ public async Task<IActionResult> Index()
             return View(model);
         }
 
+
+        public async Task<List<AadGroup>> GetSecurityGroups(string term) => await _graphService.QueryGroups(term);
+
```
Comment (emlyn-arkahna) [inline src/PowerBiAuditApp.Client/Controllers/AdminController.cs:51]: Can we decorate this with the HTTP method please

File: src/PowerBiAuditApp.Client/Controllers/AdminController.cs [hunk 2]
```diff
@@ -56,22 +60,21 @@ public async Task<IActionResult> RefreshReports()
...
         }
 
-        [HttpPost]
         public async Task<IList<ReportDetail>> SaveReportDisplayDetails(string query)
         {
-            // Example graph service call 
-            //var bob = await _graphService.GetGroupIds("City of Bunbury - Projects", "Regis Resources - Projects");
-
-            //var bill = await _graphService.QueryGroups("City");
+            // Sample error
```
Comment (emlyn-arkahna) [inline src/PowerBiAuditApp.Client/Controllers/AdminController.cs:65]: Please remove comments

File: src/PowerBiAuditApp.Client/wwwroot/js/admin.js [hunk 3]
```diff
@@ -1,5 +1,29 @@
 ﻿var original = $('#reportDisplayForm').serialize();
 
+$('.js-data-example-ajax').select2({
```
Comment (emlyn-arkahna) [inline src/PowerBiAuditApp.Client/wwwroot/js/admin.js:3]: can we change the name on this so it's meaningful

File: src/PowerBiAuditApp.Client/wwwroot/js/admin.js [hunk 4]
```diff
@@ -30,12 +58,16 @@ $('#reportDisplayForm').submit(function (event) {
...
     $.post("/Admin/SaveReportDisplayDetails", data)
         .done(() => {
-            $("#report-saved-alert").removeClass('d-none');
-            $(".view-main").scrollTop(0);
-        });
+            $('#report-saved-alert').removeClass('d-none');
+            $('.view-main').scrollTop(0);
 
-    console.log("Form submitted");
+            console.log("Form submitted");
```
Comment (emlyn-arkahna) [inline src/PowerBiAuditApp.Client/wwwroot/js/admin.js:64]: Do you mind removing the console log from the output

File: src/PowerBiAuditApp.Client/Controllers/AdminController.cs (hunk 2 above)
Comment (michaelbish) [reply src/PowerBiAuditApp.Client/Controllers/AdminController.cs:65]: Done

File: src/PowerBiAuditApp.Client/wwwroot/js/admin.js (hunk 4 above)
Comment (michaelbish) [reply src/PowerBiAuditApp.Client/wwwroot/js/admin.js:64]: Done

File: src/PowerBiAuditApp.Client/wwwroot/js/admin.js (hunk 3 above)
Comment (michaelbish) [reply src/PowerBiAuditApp.Client/wwwroot/js/admin.js:3]: Done

File: src/PowerBiAuditApp.Client/Controllers/AdminController.cs (hunk 1 above)
Comment (michaelbish) [reply src/PowerBiAuditApp.Client/Controllers/AdminController.cs:51]: Done

**Review by emlyn-arkahna — APPROVED** [review]
Approved

**Pull request merged**